website/docs/r/api_management_api_diagnostic.html.markdown

```diff
@@ -144,6 +145,8 @@ The following arguments are supported:
 
 * `log_client_ip` - (Optional) Log client IP address.
 
+* `metrics` - (Optional) Emit custom metrics via emit-metric policy. Applicable only to Application Insights diagnostic settings.
```

Comment:
if this is only applicable for Application Insights diagnostic settings, could we add some validation in the create/update to ensure it is only set when these settings are used?